content/blog/jt15s/2020-12-31_hello_from_the_haiku_promotion_team.md

```diff
@@ -0,0 +1,74 @@
+
+How will we do this, you might ask? Well, there are a couple of aspects of our strategy. 
+
+### The media and reviewers
+A database of all the reviews/mentions of Haiku is being compiled for easy reference and so we can contact a variety of media sources when we have a new release. Additionally, we are looking for more technology-related channels who may have not mentioned or reviewed Haiku before and see if they are interested in having a closer look at the OS. 
+
+In terms of the media in general, we are planning to prepare press releases for the media - a press release is simply a formal, properly formatted version of an announcement. By releasing press releases it makes it easier for other media organisations to find major Haiku announcements and run a story about Haiku. 
+
+In terms of reviewers, as mentioned before we are planning to contact reviewers and media organisations once Beta 3 is released. We have also prepared a reviewer information sheet with tips for reviewers, which is in the draft stage. 
+
+### General Users
+We mustn't forget about what is probably our main user group: general users. General users of Haiku are those who come to explore and try out the OS. Due to Haiku's remaining rough edges and still lurking bugs (we are in Beta after all), our general users are typically more understanding of the limitations and issues with the project. Attracting more general users and making sure our general userbase is content is very important.
+Strategies such as making sure user queries are answered as well as spreading the word about new apps and releases is a good first start, but we are considering various other methods too.
+
+### App Developers
+If you follow technology news, you might have seen a mention of Medo, the recently-released 4K editor clocking in at only 1.44MB! This number alone highlights one of Haiku's many development advantages: our apps are small and light - even if you have a project with lots of features crammed into it, on Haiku the file size will likely be very reasonable. Add that to the fact that we have HaikuPorts, an organised porting system, that we use C++, one of the most popular progamming languages in the world, and that we are compatible with frameworks like Qt, a popular UI frontend framework, and you get the development environment that is Haiku. We want to attract more app developers, whether that's developers who want to port their existing app over to Haiku, or new developers who wish to make an app in Haiku that they feel is needed.
+Additionally, we want to make any existing ports of FOSS projects to Haiku official platforms - this means having Haiku listed as an officially supported platform on the project's website. By getting Haiku listed on projects such as KDE, more people will be aware of Haiku and the fact that Haiku has a multitude of apps available to download.
+
+### Researchers
```

Comment:
I found mmu_man's poster about this: http://eurosys2010.sigops-france.fr/poster_demo/eurosys2010-final17.pdf

It also mentions Axel's work on the filesystem layer and Colin's work on Wifi (https://www.haiku-os.org/news/2016-02-26_wlan_master_thesis_published/) support were both done as master thesis in universities. I think this was also the case for Rudolf's NVidia driver: https://www.haiku-os.org/legacy-docs/writing-video-card-drivers/

And, maybe if we extend this to "research and education", we can also talk about GSoC and http://semesterofcode.com (we "participated" in the latter but received no student applications)

Reply:
@pulkomandy Do you have a link to Axel's work on the filesystem layer?

Reply:
No, I couldn't find it.